spec/lib/code_breaker/code_breaker_game_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe CodeBreaker::CodeBreakerGame do
+  describe '#start_game' do
+    it 'raises ValidatorError (to short)' do
+      expect { described_class.new('Vl', 'easy') }.to raise_error(CodeBreaker::ValidatorError)
+    end
+
+    it 'raises ValidatorError (to long)' do
+      expect do
+        described_class.new('VladVladVladVladVladVlad', 'easy')
```

Comment:
How can I understand why it's too long?
```suggestion
        described_class.new('A' * CodeBreaker::Constants::NAME_MAX_LENGTH.next, 'easy')
```